### ISSUE TITLE
Product images thumbnails layout responsive controls

### ISSUE THREAD

**Issue by Matlino** [soflyy/oxygen-bugs-and-features#2333]
Product images thumbnails layout responsive controls seem to not work.

Steps to reproduce:
1. Insert 'Product images' element.
2. In Primary->Layout set 'Vertically left' for 'All devices'.
3. Set 'Horizontally bottom' for 'Less than 768px'.

The result will be 'Horizontally bottom' on all devices.

**Comment by madjedo** [comment]
I don't think this is a "bug". I just don't think this is supported. Would like this supported out of the box though.
